com.unity.render-pipelines.high-definition/Documentation~/light-anchor.md

```diff
@@ -13,4 +17,5 @@ You can use a Light Anchor to light a scene in Rendered Camera Space. To use a L
 | Roll | Use the right icon to control the Rollof the light. This tool becomes gray when you move the icon. This is especially useful if the light has an IES or a Cookie. |
 | Distance | Controls the distance between the light and its anchor in world space. |
 | Up Direction | Defines the space of the up direction of the anchor. When this value is set to Local, the Up Direction is relative to the camera. |
+| Anchor Position Override | Allows to use a Gameobject's [Transform](https://docs.unity3d.com/ScriptReference/Transform.html) as anchor position instead of the LightAnchor's Transform. When this property is assigned, the Light Anchor will update it's Transform if the Transform in **Anchor Position Override** changes. |
```

Comment:
Suggestion: 

"Allows you to use a GameObject's [Transform](https://docs.unity3d.com/ScriptReference/Transform.html) as anchor position instead of the LightAnchor's Transform. When the Transform of the GameObject you assigned to this property changes, the Light Anchor's Transform also changes."